README.md

```diff
@@ -37,15 +37,10 @@ If you don't have a favorite method already, or you don't know what those words
     - Start with the built-in `Terminal` application.
     - [Learn the Mac OS X Command Line][]
 - For Windows Users:
-    - Use the native [Command Prompt][]
-    - [Open a command prompt in a folder (Windows)][]
-    - [Running executables from the command line (Windows)][]
-    - To show emoji in Windows Commmand Prompt: `chcp 65001` and choose a font with emoji support (Consolas is one).
-
-[Learn the Mac OS X Command Line]: https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line
```

Comment:
This line (MacOS X Command Line link) needs to be preserved

Reply:
oops, thanks

Reply:
This link is still missing. It was added to README-WINDOWS.md (where it is not needed) instead of being preserved here, where it is needed.

Reply:
oops. fixed for real this time, sorry. thanks for catching